Mathlib/RingTheory/LocalProperties.lean

```diff
@@ -440,7 +440,9 @@ theorem IsLocalization.smul_mem_finsetIntegerMultiple_span [Algebra R S] [Algebr
   rw [Submodule.span_smul] at hx₁
   replace hx : _ ∈ y' • Submodule.span R (s : Set S') := Set.smul_mem_smul_set hx
   rw [hx₁] at hx
-  erw [← g.map_smul, ← Submodule.map_span (g : S →ₗ[R] S')] at hx
+  -- ACL : coercion does not work anymore ???
+  -- erw [← g.map_smul, ← Submodule.map_span (g : S →ₗ[R] S')] at hx
+  erw [← g.map_smul, ← Submodule.map_span g.toLinearMap] at hx
```

Comment:
I'll have to think a bit more about the (`NonUnital`)`AlgHom` coercions not being found, this seems somehow to only occur in complicated applications and not in simple testcases. Presumably there is some parameter that is underdetermined in this rewrite and that is blocking the coercion being found.